### PR TITLE
execute LOL with cmd, to avoid new game process have a parent

### PR DESCRIPTION
**Pull request by vicmaeg**

### CODE REVIEW

File: source/Playnite.Common.NetStandard/ProcessStarter.cs [hunk 1]
```diff
@@ -58,6 +58,42 @@ public static Process StartProcess(string path, string arguments, string workDir
...
+
+            var info = new ProcessStartInfo
+            {
+                FileName = "cmd",
+                Arguments = $"/C start {gameExe} {arguments}",
+                WorkingDirectory = string.IsNullOrEmpty(workDir) ? directory : workDir,
+                WindowStyle = ProcessWindowStyle.Hidden
+            };
+
+            if (asAdmin)
```
Comment (SandraPG85) [inline source/Playnite.Common.NetStandard/ProcessStarter.cs:91]: why is this necessary?

**Review by SandraPG85 — CHANGES_REQUESTED** [review]

File: source/Playnite.Common.NetStandard/ProcessStarter.cs (hunk 1 above)
Comment (vicmaeg) [reply source/Playnite.Common.NetStandard/ProcessStarter.cs:91]: The API of playnite has an argument asAdmin, if you want to launch the game as administrator if is installed on a path that has admin permissions, normally this shouldn't be necessary, and it is called always as asAdmin = false, but I mantain this option.

**Review by SandraPG85 — APPROVED** [review]
